### ISSUE TITLE
help(read_line_to_string) missing

### ISSUE THREAD

**Issue by erlanger** [SWI-Prolog/swipl-devel#448]
```
3 ?- h(read_line_to_string).
Warning: No help for read_line_to_string.
Warning: Use ?- apropos(query). to search for candidates.
true.
```

**Comment by JanWielemaker** [comment]
library(readutil) appears to use old LaTeX docs rather than PlDoc.  The content of man/lib/readutil.doc should partly be merged into library/readline.pl and next the CMakeLists.txt should be adapted to use PlDoc for this library.

**Comment by JanWielemaker** [comment]
Fixed with ed1052ed3c97933f0f9096ffe149fe2ff38fbcd8

**Comment by erlanger** [comment]
Thanks!
